lib/reporters/base.js

```diff
@@ -231,13 +235,16 @@ exports.list = function(failures) {
  * of tests passed / failed etc.
  *
  * @param {Runner} runner
+ * @param {Object} options runner optional options
  * @api public
  */
 
-function Base(runner) {
+function Base(runner, options) {
   var stats = this.stats = { suites: 0, tests: 0, passes: 0, pending: 0, failures: 0 };
   var failures = this.failures = [];
 
+  this.reporterOptions = options ? options.reporterOptions : null;
```

Comment:
`output` was the name used previously by the `xunit` reporter, I preferred not changing it to keep backward compat.

Also, since `reporterOptions` are in principle reporter specific, I don't know if some other reporter might decide to interpret (or it's already interpreting!) `output` in a different way. That's why I went by making each interested reporter call the purely helper method `openOutput` explicitly.